src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -696,15 +701,17 @@ export class CalciteColorPicker {
 
     const valueIsNoColor = allowEmpty && !value;
     const parsedMode = parseMode(value);
-    const valueIsCompatible = (format === "auto" && parsedMode) || format === parsedMode;
+    const valueIsCompatible =
+      valueIsNoColor || (format === "auto" && parsedMode) || format === parsedMode;
```

Comment:
nitpick. Maybe const could be `valueNotAColor` or `colorlessValue`? no biggie. Just a weird name `valueIsNoColor`

Reply:
Good idea. Changed it to `willSetNoColor` to better express the intent.